sdcm/nemesis.py

```diff
@@ -1830,14 +1830,23 @@ def repair_nodetool_rebuild(self):
             self.target_node.run_nodetool('rebuild', retry=0)
 
     def disrupt_nodetool_cleanup(self):
-        # This fix important when just user profile is run in the test and "keyspace1" doesn't exist.
         test_keyspaces = self.cluster.get_test_keyspaces()
-        for node in self.cluster.nodes:
+
+        # Inner disrupt function for ParallelObject
+        def _dis_nodetool_cleanup(node):
+            # This fix important when just user profile is run in the test and "keyspace1" doesn't exist.
```

Comment:
This comment is related to the `test_keyspaces = self.cluster.get_test_keyspaces()` line.
So, need to return it back too.

Reply:
My bad... Done.